pkg/object/validation/collector.go

```diff
@@ -14,14 +14,14 @@ import (
 // extracting the IDs of the invalid objects.
 type Collector struct {
 	Errors     []error
-	InvalidIds object.ObjMetadataSet
+	InvalidIds object.ObjMetadataSet //nolint:revive
```

Comment:
InvalidIDs?

Reply:
Same as in another place - this is an exported field, didn't want to make a breaking change. WDYT?